docs/_guides/codewind-eclipse-quick-guide.md

```diff
@@ -142,12 +140,12 @@ public class Calculator extends Application {
     }
 }
 ```
-Any changes you make to your code are automatically built and re-deployed by Codewind and you can view them in your browser.
+Any changes that you make to your code are automatically built and redeployed by Codewind and you can view them in your browser.
```

Comment:
This is a compound sentence. Add a comma after "Codewind" because you need a comma + a conjunction when separating two independent clauses.